src/main/resources/application-test.properties

```diff
@@ -0,0 +1,15 @@
+#test environment

```

Comment:
this file should be under test folder instead of main
and based on default configuration this would work